From 45063cdcd06a89ddfdcf17fe33d9262023a7163f Mon Sep 17 00:00:00 2001
From: Justin Schneck <j.schneck@peridio.com>
Date: Fri, 21 Nov 2025 20:19:43 -0500
Subject: [PATCH] Remove man page creation

---
 meson.build | 21 ---------------------
 1 file changed, 21 deletions(-)

diff --git a/meson.build b/meson.build
index daeaccc..57e17dd 100644
--- a/meson.build
+++ b/meson.build
@@ -490,27 +490,6 @@ configure_file(
     output : 'config.h',
     configuration : cdata)
 
-# Generate a manpage from the POPT documentation
-help2man = find_program('help2man')
-
-manpage = custom_target('manpage',
-    output : 'sscg.8',
-    capture : true,
-    command : [
-        help2man,
-        '-s', '8',
-        '-n', 'Tool for generating x.509 certificates',
-        '-N',
-        sscg,
-    ],
-    install : true,
-    build_by_default : true,
-    install_dir : join_paths(
-        get_option('prefix'),
-        get_option('mandir'),
-        'man8'),
-)
-
 # Internationalization support
 if gettext.found()
     subdir('po')
-- 
2.43.0

